spec/spec_helper_acceptance.rb

```diff
@@ -35,6 +35,11 @@
 
     hosts.each do |host|
       on host, puppet('module', 'install', 'puppetlabs-stdlib'), acceptable_exit_codes: [0, 1]
+
+      if fact('osfamily') == 'Debian' && fact('operatingsystemmajrelease') == '8'
```

Comment:
I think there should be a comment above, like "work around issue with openhpid not existing".

Also, switch to structured facts, so `fact('os.family')`, I believe, and `fact('os.release.major')`?

Reply:
Oh wait, this is still Puppet 3 compatible, so I guess leave unstructured for now.

Reply:
Comment:
      # For Debian 8 "jessie", we need
      # - pacemaker and crmsh delivered in jessie-backports only
      # - openhpid post-install may fail (https://bugs.debian.org/785287)